### PR TITLE
sanitize topics

### PR DESCRIPTION
**Pull request by paulwe**

### CODE REVIEW

File: helpers.go [hunk 1]
```diff
@@ -17,48 +16,85 @@ func newStreamID() string {
...
 	}
+	return buf
 }
 
-func getHandlerKey(rpc string, topic []string) string {
-	if len(topic) > 0 {
-		return fmt.Sprintf("%s|%s", rpc, strings.Join(topic, "|"))
-	} else {
-		return rpc
+func formatChannel(prefix0, prefix1 string, topics []string, suffix string) string {
```
Comment (frostbyte73) [inline helpers.go:50]: we can just treat clientID as a topic instead of prefix1 here

Comment (frostbyte73) [reply helpers.go:50]: same with suffix

Comment (frostbyte73) [reply helpers.go:50]: actually it could just be `formatChannel(parts ...string)`

**Review by frostbyte73 — APPROVED** [review]

File: helpers.go (hunk 1 above)
Comment (paulwe) [reply helpers.go:50]: resolved out of band

**Pull request merged**